doc/introduction/operations.rst

```diff
@@ -137,6 +137,30 @@ Qubit observables
 
 :html:`</div>`
 
+Grouping Pauli words
+^^^^^^^^^^^^^^^^^^^^
+
+Grouping Pauli words can be used for the optimizing the measurement of qubit
+Hamiltonians. Along with groups of observables, post measurement rotations can
+also be obtained using :func:`~.optimize_measurements`:
+
+.. code-block:: python
+
+    >>> obs = [qml.PauliY(0), qml.PauliX(0) @ qml.PauliX(1), qml.PauliZ(1)]
+    >>> coeffs = [1.43, 4.21, 0.97]
+    >>> post_rotations, diagonalized_groupings, grouped_coeffs = optimize_measurements(obs, coeffs)
+    >>> post_rotations
+    [[RY(-1.5707963267948966, wires=[0]), RY(-1.5707963267948966, wires=[1])],
+     [RX(1.5707963267948966, wires=[0])]]
+
+The post measurement rotations can be used to diagonalize the partitions of
+observables found.
+
+For further details on measurement optimization, graph coloring to be solved
+for grouping observables and auxiliary functions refer to the
+:doc:`/code/qml_grouping` subpackage.
```

Comment:
:+1: This looks great @antalszava. HIghlights that this functionality exists, but keeps it short, sweet, and succinct, which is how a quickstart should read (the quickstarts are _reference guides_, first and foremost, not tutorials).

If a user would like more information on the grouping functions, they can then refer to the `qml.grouping` module page.